common4j/src/main/com/microsoft/identity/common/java/platform/JweResponse.java

```diff
@@ -89,9 +91,13 @@ public String getAuthenticationTag() {
     }
 
     public static JweResponse parseJwe(String jwe) throws JSONException {
+        Span span = Span.current();
```

Comment:
nit: final

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1847/commits/9bc9320913bc03bd1a733bdf0fd5e8442caafae0